doc/source/notebooks/tailor/external-mean-function.pct.py

```diff
@@ -227,16 +268,17 @@ def mean_squared_error(y, y_pred):
     mean_squared_errors.append(mse)
     plt.title(f"Test Task {i + 1} | MSE = {mse}")
     plt.legend()
+    plt.show()
```

Comment:
If we don't close the `plt.figure` explicitly, if we try to plot too many, we get this warning:
```<ipython-input-38-510b3796bcb3>:6: RuntimeWarning: More than 20 figures have been opened. Figures created through the pyplot interface (`matplotlib.pyplot.figure`) are retained until explicitly closed and may consume too much memory. (To control this warning, see the rcParam `figure.max_open_warning`).
```